package.json

```diff
@@ -83,6 +88,8 @@
     "test:webserver": "grunt test:webserver",
     "concat": "grunt concat",
     "build": "grunt build",
-    "requirejs": "grunt requirejs"
+    "requirejs": "grunt requirejs",
+    "lint": "eslint nodejs/**/*.js common/**/*.js browser/lib/**/*.js",
```

Comment:
You can also add one more command for ```lint:fix``` using --fix flag for fixing possible linting errors.